Gruntfile.js

```diff
@@ -70,6 +70,14 @@ module.exports = function(grunt) {
         files: {
           'tmp/remove_attrs': ['test/fixtures/mail.svg', 'test/fixtures/lock.svg']
         }
+      },
+      preserve_viewbox: {
+        options: {
+          preserveViewbox: true
```

Comment:
Can we camelCase this option to `preserveViewBox`? Otherwise your code seems great, looking forward to publish an update 👍